electron/updater.ts

```diff
@@ -196,7 +196,7 @@ const registerExecuteProcessEvent = (rendererPath: string, executablePath: strin
 const getArtifactUrl = (launcherPaths: LauncherPaths) => {
   if (isUsingRollout()) {
     // Rollout
-    return `https://github.com/decentraland/explorer-desktop/releases/download/${remoteVersion}${launcherPaths.artifactUrl}`
+    return `https://cdn.decentraland.org/@dcl/explorer-desktop/${remoteVersion}${launcherPaths.artifactUrl}`
```

Comment:
When creating URLs, always sanitize components using `encodeURIComponent`. For this change, the `launcherPaths.artifactUrl` or `remoteVersion` should not include any `"/"`

```suggestion
    return `https://cdn.decentraland.org/@dcl/explorer-desktop/${encodeURIComponent( remoteVersion)}${encodeURIComponent(launcherPaths.artifactUrl)}`
```